coderd/database/migrations/000112_template_disable_user_scheduling.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE "templates"
+	ADD COLUMN "allow_user_auto_start" boolean DEFAULT true NOT NULL,
+	ADD COLUMN "allow_user_auto_stop" boolean DEFAULT true NOT NULL;
```

Comment:
You won't like this comment... In the codebase, we're using "autostart" and "autostop".

Reply:
nothing a sed can't fix :^)